src/interfaces.rs

```diff
@@ -291,6 +291,16 @@ pub fn dispatch(
             }
         }
 
+        OpCode::StartProcess => plugin.start_process(),
+        OpCode::StopProcess => plugin.stop_process(),
+
+        OpCode::GetNumMidiInputs => {
+            return plugin.get_info().midi_inputs as isize;
```

Comment:
Better to use the cached version of info: `(*effect).get_cache().info.midi_inputs`
Same with outputs.